docs/storage_change_status.md

```diff
@@ -0,0 +1,138 @@
+# Storage change status
+
+> Analysis of possible storage change statuses in [EIP-2200] / [EIP-1283].
+
+## Cost constants
+
+| Constant | Value |
+| -------- | ----- |
+| SLOAD    | 200   |
```

Comment:
You're using `200` for all of these - but it's `800` no?

Reply:
Yes, the **unmerged** EIP-2200 uses 800, but I wasn't sure the repricing 100% confirmed to go though. The precise value is secondary in the context of this document.

Reply:
It did go through, eventually, haha :)